migrations/postgres/20220705072248_create_tables.up.sql

```diff
@@ -43,7 +43,8 @@ CREATE TABLE physical_partition_column (
     name VARCHAR NOT NULL,
     type VARCHAR NOT NULL,
     min_value BYTEA,
-    max_value BYTEA
+    max_value BYTEA,
+    null_count INTEGER
```

Comment:
At some point we'll have to start using actual migrations here, since currently editing the same file means we'll have to recreate existing instances from scratch. Nbd right now though.

Reply:
Yup, I realize that, thanks. I figured this would happen only after the initial release, right?